pom.xml

```diff
@@ -115,4 +133,23 @@
       <timezone>+8</timezone>
     </developer>
   </developers>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <configuration>
+          <rules combine.children="append">
+            <requireSameVersions>
+              <dependencies>
+                <dependency>net.java.dev.jna:jna</dependency>
+                <dependency>net.java.dev.jna.extensions:jna-platform</dependency>
+              </dependencies>
+            </requireSameVersions>
+          </rules>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
```

Comment:
If this Enforcer check ever starts failing, then add a `<dependencyManagement>` entry for `jna-platform` matching the version delivered by core.